### PR TITLE
conformance: Gloo Gateway report for v1.0.0 

### PR DESCRIPTION
**Pull request by EItanya**
**What type of PR is this?**

/kind documentation
/area conformance

**What this PR does / why we need it**:

This PR adds a conformance report for gloo gateway against gateway API v1.0.0

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE

```

### CODE REVIEW

**Comment by robscott** [comment]
Thanks @EItanya! I'd recommend updating implementations.md at the same time and adding a conformance badge to link to this report.

/ok-to-test
/approve

**Comment by EItanya** [comment]
> Thanks @EItanya! I'd recommend updating implementations.md at the same time and adding a conformance badge to link to this report.
> 
> /ok-to-test /approve

Sounds good, will do!

File: site-src/implementations.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Implementors and integrators of Gateway API are encouraged to update this docume
 - [Emissary-Ingress (Ambassador API Gateway)][4] (alpha)
 - [Envoy Gateway][18] (beta)
 - [Flomesh Service Mesh][17] (beta)
-- [Gloo Edge 2.0][5] (work in progress)
+- [Gloo Gateway 2.0][5] (beta)
```
Comment (sunjayBhatia) [inline site-src/implementations.md:20]: you can also add to the Gloo section to get a badge with something like this: https://github.com/kubernetes-sigs/gateway-api/blob/1ca2a9e3ffa51219e85aa3badde220cca7c3ad30/site-src/implementations.md?plain=1#L159

**Review by sunjayBhatia — COMMENTED** [review]
looks good, but holding off approving until we get confirmation if @EItanya wants to add a badge or not

File: conformance/reports/v1.0.0/gloo-gateway.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      Failed: 0
+      Passed: 29
+      Skipped: 0
+    summary: ""
+  extended:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 6
+      Skipped: 0
```
Comment (sunjayBhatia) [inline conformance/reports/v1.0.0/gloo-gateway.yaml:26]: yeah this is one of those reports where skipped is not counted yet, but should be fixed soon

**Comment by EItanya** [comment]
> looks good, but holding off approving until we get confirmation if @EItanya wants to add a badge or not

Oh now I see what you mean, I can add that!

File: site-src/implementations.md [hunk 3]
```diff
@@ -226,11 +226,14 @@ FSM support of Gateway API is built on top [Flomesh Gateway API](fgw) and it cur
...
 
-### Gloo Edge
+### Gloo Gateway
 
-Gloo Edge 2.0 is an Istio-native, fully-featured Envoy based API gateway that brings [Gloo Edge][glooedge] functionality to Istio. The [Solo.io][solo] team is actively working towards an implementation of the Gateway API.
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.0.0-GlooGateway-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/gloo-gateway.yaml)
 
-[glooedge]:https://docs.solo.io/gloo-edge/
+[Gloo Gateway][gloogateway] by [Solo.io][solo] is a feature-rich, Kubernetes-native ingress controller and next-generation API gateway.
+Gloo Gateway 2.0 brings the full power and community support of the Gateway API to it's existing control-plane implementation.
```
Comment (robscott) [inline site-src/implementations.md:234]: Very tiny nit:

```suggestion
Gloo Gateway 2.0 brings the full power and community support of Gateway API to it's existing control-plane implementation.
```

x-ref: https://github.com/kubernetes-sigs/gateway-api/issues/2553

**Review by robscott — COMMENTED** [review]
Thanks @EItanya! One last very tiny nit but otherwise LGTM.

File: site-src/implementations.md (hunk 3 above)
Comment (EItanya) [reply site-src/implementations.md:234]: If only programing also improved my grammar 😆, thank you @robscott

**Comment by robscott** [comment]
Thanks!

/lgtm

**Pull request merged**